frontend/src/app/features/auth/verify-email/verify-email.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="verify-email-wrapper">
+    <h1>Finish registration</h1>
```

Comment:
I'd say you need a localization. Add localization.

Reply:
Added localization possibility by installing appropriate package and adding attribute, used for translation, to all text-containing elements